src/components/Nav/Nav.jsx

```diff
@@ -1,7 +1,123 @@
-import React from 'react';
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { Link } from 'react-router-dom';
 
 const Nav = () => {
-  return <div>Nav</div>;
+  const [userInput, setUserInput] = useState('');
+
+  const getInput = e => {
+    setUserInput(e.target.value);
+  };
+
+  return (
+    <WrapperNav>
+      <NavTop>
+        <StyledLinkLogo to="/">
+          <LogoImg src="../images/BaBee.png" />
+        </StyledLinkLogo>
+        <SearchDiv>
+          <SearchImg src="../images/Nav/search.png" />
+          <SearchInput placeholder="상품명 입력" onChange={getInput} />
+        </SearchDiv>
+        <LinkDiv>
+          <StyledLinkRight to="/productsell">판매하기</StyledLinkRight>
+          <LinkBar>|</LinkBar>
+          <StyledLinkRight to="/mypage">마이페이지</StyledLinkRight>
+          <LinkBar>|</LinkBar>
+          <StyledLinkRight to="/babeeTalk">베비톡</StyledLinkRight>
+        </LinkDiv>
+        <BaBeeTalkImg src="../images/Nav/Comma.png" />
+      </NavTop>
+      <NavBottom>
+        <StyledLinkCate to="/">내의/속옷</StyledLinkCate>
+        <StyledLinkCate to="/">상의/하의</StyledLinkCate>
+        <StyledLinkCate to="/">원피스</StyledLinkCate>
+        <StyledLinkCate to="/">신발</StyledLinkCate>
+        <StyledLinkCate to="/">악세사리</StyledLinkCate>
+        <StyledLinkCate to="/">이불/침구</StyledLinkCate>
+        <StyledLinkCate to="/">인형/모빌</StyledLinkCate>
+      </NavBottom>
+    </WrapperNav>
+  );
 };
 
+const WrapperNav = styled.div`
+  position: sticky;
+  height: 146px;
+  padding-top: 35px;
+  border-bottom: 1px solid ${props => props.theme.whiteGreyD9};
+`;
+const NavTop = styled.div`
+  width: 100%;
+
+  display: flex;
+  justify-content: center;
+`;
+const LogoImg = styled.img`
+  width: 95px;
```

Comment:
```suggestion
const WrapperNav = styled.div`
  position: sticky;
  height: 146px;
  padding-top: 35px;
  border-bottom: 1px solid ${props => props.theme.whiteGreyD9};
`;

const NavTop = styled.div`
  width: 100%;
  display: flex;
  justify-content: center;
`;

const LogoImg = styled.img`
  width: 95px;
```
가독성을 위해 한칸씩 띄워서 작성해주세요!